lms/djangoapps/certificates/views.py

```diff
@@ -481,9 +468,19 @@ def render_html_view(request, user_id, course_id):
     configuration = CertificateHtmlViewConfiguration.get_config()
     context.update(configuration.get('default', {}))
 
+    # Translators:  This is the copyright line which appears at the bottom of the certificate page/screen
+    context['copyright_text'] = _('&copy; {year} {platform_name}. All rights reserved.').format(
+        year=datetime.now().year,
+        platform_name=settings.PLATFORM_NAME
+    )
+
     # Translators:  This text is bound to the HTML 'title' element of the page and appears
     # in the browser title bar when a requested certificate is not found or recognized
     context['document_title'] = _("Invalid Certificate")
+    context['company_tos_urltext'] = _("Terms of Service &amp; Honor Code")
```

Comment:
This should either be `_("Terms of Service & Honor Code")`, or you should put in a translators comment explaining that the the `&amp;` is a stand-in for an ampersand, and should not be translated.